go/obscuronode/enclave/sql/edbattestation.go

```diff
@@ -0,0 +1,204 @@
+package sql
+
```

Comment:
Aestetic comment: file name `edb_attestation.go` instead of `edbattestation.go`

Reply:
Yeah much prefer that, had in my head there was a go convention of only using underscores in test filenames but that doesn't seem to be a thing :)